sys/zuse.hoon

```diff
@@ -7942,4 +7936,148 @@
       --
     --
   --
+:: |dawn: pre-boot request/response de/serialization and validation
+::
+++  dawn
+  =>  |%
+      :: +live: public network state of a ship
+      ::
+      +$  live  (unit [=life breach=?])
+      :: +seed: private boot parameters
+      ::
+      +$  seed  [who=ship lyf=life key=ring sig=(unit oath:pki:jael)]
+      --
+  |%
+  :: |give:dawn: produce requests for pre-boot validation
+  ::
+  ++  give
+    =/  tract  ships:contracts:constitution:ethe
+    |%
+    :: +czar:give:dawn: Eth RPC for galaxy table
+    ::
+    ++  czar
+      ^-  octs
+      %-  as-octt:mimes:html
+      %-  en-json:html
+      %-  batch-read-request:ethereum
+      %+  turn  (gulf 0 255)
+      |=(gal=@ [`(scot %ud gal) tract ['getKeys(uint32)' [%uint gal]~]])
+    :: +hull:give:dawn: Eth RPC for ship's contract state
+    ::
+    ++  hull
+      |=  who=ship
+      ^-  octs
+      %-  as-octt:mimes:html
+      %-  en-json:html
+      (read-request:ethereum [~ tract ['ships(uint32)' [%uint `@`who]~]])
+    :: +turf:give:dawn: Eth RPC for network domains
```

Comment:
Comment style consistency please! (:

Reply:
Likewise, `|foo` for a library core. `+bar:foo` for a gate in said core (`+bar` vs `+bar:foo` is still subject to debate, I believe).

Reply:
Here I was referring to the lack of below-comment whiteline specifically, not the `+` vs `|`. I should've been more explicit!

Reply:
Fixed